src/forms/auth/register.tsx

```diff
@@ -213,8 +194,15 @@ class RegistrationForm extends React.Component<
   }
 }
 
-const WrappedRegistrationForm = Form.create()(RegistrationForm);
+const WrappedRegistrationForm = Form.create()(withRouter(RegistrationForm));
 
+function addProps(RegForm: any) {
+  return class extends React.Component<RegistrationFormSubmitProps> {
+    render() {
+      return <RegForm {...this.props} />;
+    }
+  };
+}
```

Comment:
had to add this function thingy to allow props to be passed into the WrappedRegistrationFormComponent

Reply:
janky but it works!